components/automate-ui/src/app/pages/data-feed/data-feed.component.scss

```diff
@@ -5,51 +5,39 @@
   margin-top: 80px;
 }
 
-.filter-button chef-button {
-  margin-left: 0;
-}
-
 chef-button:first-child {
-  margin-top: 0;
-  margin-left: 0;
-}
-
-chef-icon#sort-desc {
-  color: $chef-primary-dark;
-
-  &::after {
-    color: $chef-dark-grey;
-  }
-}
-
-chef-icon#sort-asc {
-  color: $chef-dark-grey;
-
-  &::after {
-    color: $chef-primary-dark;
-  }
+  display: flex;
+  float: right;
+  align-items: center;
+  margin-top: -360px;
 }
 
 .actions {
   padding: 8px 1em;
 
   chef-button[icon] {
-    min-height: 32px;
     font-size: 18px;
     line-height: 32px;
-
+    
```

Comment:
check `npm lint`
in `components/automate-ui` directory

Reply:
Done